collect/stressRelief.go

```diff
@@ -119,25 +179,62 @@ func (s *StressRelief) Start() error {
 	return nil
 }
 
-func (s *StressRelief) UpdateFromConfig(cfg config.StressReliefConfig) error {
+type stressReliefMessage struct {
+	level  uint
+	peerID string
+}
+
+func newStressReliefMessage(level uint, peerID string) *stressReliefMessage {
+	return &stressReliefMessage{level: level, peerID: peerID}
+}
+
+func (msg *stressReliefMessage) String() string {
+	return msg.peerID + ":" + fmt.Sprint(msg.level)
+}
+
+func unmarshalStressRelifMessage(msg string) (*stressReliefMessage, error) {
+	if len(msg) < 2 {
+		return nil, fmt.Errorf("empty message")
+	}
+
+	parts := strings.SplitAfter(msg, ":")
```

Comment:
I think you want Split, not SplitAfter -- SplitAfter is going to keep the : which I don't think you want.

Reply:
Good catch! I switched to SplitN since we already have the assumption that the message should only contain 2 parts separated by :